src/components/identity/IdentityLocRequest.css

```diff
@@ -0,0 +1,27 @@
+.IdentityLocRequest .SelectLegalOfficer {
+    margin: 10px 20px 10px 10px;
+    text-align: left;
+    min-width: 420px;
+}
+
+.IdentityLocRequest .SelectLegalOfficer .drop-down {
```

Comment:
This rule should be moved to `SelectedLegalOfficer.css`.

Reply:
No. This breaks the current layout of Protection request, which also includes the componnent `SelectLegalOfficer`.

Reply:
OK, but this is nonetheless a very bad practice (to have CSS styling specific to a child component defined in the stylesheet of a parent component). In that case, it is better, if possible, to provide some property to the component and let it apply conditionally some style in function of the property value.

Reply:
Got it. This commit will be anyway removed, following the decision to convert the dialog into plain page.